test/JDBC/input/BABEL-733-vu-prepare.mix

```diff
@@ -0,0 +1,44 @@
+-- tsql
+CREATE DATABASE babel_733_db1;
```

Comment:
Also add objects with lower/upper/mixed case names

Reply:
The function OBJECT_SCHEMA_NAME takes Oid of object and database as parameters, so the name of the object/database wouldn't make a difference.